kuka_rox_hardware_interface/src/robot_manager_node.cpp

```diff
@@ -321,6 +327,12 @@ bool RobotManagerNode::onControlModeChangeRequest(int control_mode)
     return false;
   }
 }
+
+bool RobotManagerNode::onRobotModelChangeRequest(std::string robot_model)
```

Comment:
lgtm, but why does this have to return a bool?

Reply:
this is the callback for a parameter change, so yes